aircan/dependencies/utils.py

```
@@ -72,6 +72,42 @@ def days_ago(n, hour=0, minute=0, second=0, microsecond=0):
+            request_data.update({'error': {
+                'message' : status_dict.get('error', '')
+            }})
+
+        url = urljoin(site_url, '/api/3/action/aircan_status_update')
+        response = requests.post(url,
+                        data=json.dumps(request_data),
+                        headers={'Content-Type': 'application/json',
+                                'Authorization': ckan_api_key})
+        print(response.text)
+        if response.status_code == 200:
+            resource_json = response.json()
```

Ruff (0.12.2)

Local variable `resource_json` is assigned to but never used

Remove assignment to unused variable `resource_json`

(F841)